README.md

```diff
@@ -1,223 +1,189 @@
-# Command-line test framework
-
-*Icetea* is an [mbed](www.mbed.com) test framework written
-with python 2.7. It is generally used to verify the ARM mbed
-IoT Device Platform provides the operating system and cloud services.
-
-Python 3 is preliminarily supported, but there might be some
-compatibility issues that will be fixed as they are found.
-
-When testing [`mbed OS`](https://www.mbed.com/en/platform/mbed-os/)
-*Icetea* allows you to execute commands remotely via
-the command line interface in board (`DUT`).
-The interface between the test framework and `DUT` can be
-for example UART, sockets or for example stdio (process `DUT`).
-
-A more detailed description of the *Icetea* concept is
-available [here](doc/README.md).
-
-## Installation
-
-`> python setup.py install`
-
-When installing Icetea, installing to a
-[virtualenv](https://virtualenv.pypa.io/en/stable/installation/) is
-a good idea.
-
-### Required dependencies:
-
-* In case you want to use **local mbed devices**
-  * [mbed-ls](https://github.com/armmbed/mbed-ls)
-    * To enumerate connected boards
-  * [mbed-flasher](https://github.com/ARMmbed/mbed-flasher)
-    * to flash devices automatically
-  * Both of these should be installed automatically by
-  Icetea installation.
-
-**OS specific:**
-
-Linux:
-* python-dev and python-lxml
-`sudo apt-get install python-dev python-lxml`
-* For documentation install sphinx:
-`apt-get install python-sphinx`
-
-OS X:
-* [XCode developer tools](http://osxdaily.com/2014/02/12/install-command-line-tools-mac-os-x/)
-* [MacPorts](https://www.macports.org/install.php)
-* lxml as described
-[here](http://lxml.de/installation.html#installation):
-`STATIC_DEPS=true sudo pip install lxml`
-* For documentation install sphinx:
-`sudo port install py27-sphinx`, `sudo port select --set python python27`,
-`sudo port select --set sphinx py27-sphinx`
-
-Windows:
-* python-lxml installation is problematic on Windows since
-it usually requires build tools. It can however be installed
-from pre-built binaries.
-    * Download binary for you system from the internet.
-    * Navigate to the directory where you downloaded the
-    binary and install it with `pip install <insert_file_name>`
-* For documentation install sphinx with pip:
-`pip install sphinx`
-
-### Optional dependencies
+## Icetea test framework
+
+Icetea is an [Mbed](https://www.mbed.com) test framework written
+with python. It can be used to verify the ARM mbed
+IoT Device Platform, Mbed OS and Pelion Device Management.
```

Comment:
- python -> Python
- get rid of ARM mbed... -> Pelion Device Management = former Mbed Cloud
- What does the tool actually test? This statement does not really tell anything. Tell precisely what it is for.